doc/api/stream.md

```diff
@@ -1033,10 +1033,11 @@ available, [`stream.read()`][stream-read] will return that data.
 const readable = getReadableStreamSomehow();
 readable.on('readable', function() {
   // There is some data to read now.
-  let data;
+  let data = this.read();
 
-  while (data = this.read()) {
+  while (data) {
     console.log(data);
+    data = this.read();
```

Comment:
Does ESLint not allow assignment in conditions with parentheses? I would strongly prefer `while ((data = this.read()) != null)` over duplicating code. That's what I personally use.

Reply:
Yes, the rule permits that approach too. Happy to switch to that.